fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/BankSearchService.java

```diff
@@ -22,16 +20,21 @@
 public class BankSearchService {
 
     private final TppBankSearchClient tppBankSearchClient;
-
-    @Autowired
-    RestRequestContext restRequestContext;
-
-    @Autowired
-    private TppProperties tppProperties;
+    private final RestRequestContext restRequestContext;
 
     @SneakyThrows
     public InlineResponse2001 searchBank(String keyword, Integer start, Integer max) {
-        BankSearchResponse bankSearchResponse = tppBankSearchClient.bankSearchGET(tppProperties.getFintechID(), UUID.fromString(restRequestContext.getRequestId()), keyword, start, max).getBody();
+        UUID xRequestId = UUID.fromString(restRequestContext.getRequestId());
+
+        BankSearchResponse bankSearchResponse = tppBankSearchClient.bankSearchGET(
+                xRequestId,
+                keyword,
+                null,
```

Comment:
Please add a comment about why the values are null, or add some constant like:
`public static final LocalDateTime COMPUTE_X_TIMESTAMP_UTC = null`
and use it here instead of null

Reply:
3 constants were added, with comments regarding the meaning of null values